docs/AddingNewProjects.md

```diff
@@ -0,0 +1,31 @@
+# Adding New Projects To The Repo
+
+Sample PR of final result: https://github.com/dotnet/aspnetcore/pull/41945
+
+## Creating a new project
+1. Create a new folder that will house your `.csproj` and other project-related files.
+2. (EXTREMELY IMPORTANT) Inside our new folder make a new folder called `src`.
+3. In VS, Add a new `Solution Folder` in the new `src` folder from the previous step.
```

Comment:
I think you want to add the solution folder from the folder you created in step 1, so that it will also include `tests` or `samples` you add there - can someone else confirm (@BrennanConroy @halter73 @Tratcher)? I realized recently I've been doing this wrong, I had manually been adding each .csproj (src, test, sample) one-by-one.

Reply:
I mean that this step says to add the solution folder in `src/Middleware/WebTransport/src/` when it should actually be in `src/Middleware/WebTransport/`

Reply:
Oh, ok. Just pushed the change. Thanks